<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfessionalWeatherAppBestWeatherAppEver</class>
 <widget class="QMainWindow" name="ProfessionalWeatherAppBestWeatherAppEver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search a location by postal code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>671</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Will I Need a Jacket?</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="responseField">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>731</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="disclaimer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*p.s. if it starts randomly crashing it's because my API was used up for today, try again tomorrow.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
